app/scripts/metamask-controller.js

```diff
@@ -7910,7 +7910,11 @@ export default class MetamaskController extends EventEmitter {
           this.controllerMessenger,
           'NetworkController:getNetworkClientById',
         ),
-        startTrace: trace,
+        startTrace: (options) => {
+          // eslint-disable-next-line no-unused-vars
+          const { _isStandaloneSpan, ...result } = trace(options);
```

Comment:
This is a fix for Firefox, since `_isStandaloneSpan` is undefined which isn't valid JSON. On Chrome it's stripped automatically, so I didn't catch this before.